assets/model_monitoring/components/src/genai_token_statistics_compute_metrics/constants.py

```diff
@@ -16,7 +16,9 @@
 MODEL_COMPLETION_COUNT = "model_completion_count"
 
 # Constants for attribute keys
-COMPLETION_COUNT_KEY = "llm.token_count.completion"
-PROMPT_COUNT_KEY = "llm.token_count.prompt"
-TOTAL_COUNT_KEY = "     llm.token_count.total"
-MODEL_KEY = "llm.model"
+COMPLETION_COUNT_KEY = ["llm.token_count.completion", "llm.usage.completion_tokens"]
+PROMPT_COUNT_KEY = ["llm.token_count.prompt", "llm.usage.prompt_tokens", "embedding.token_count.prompt",
```

Comment:
I believe we should put the latest schema, I suppose it is "embedding.usage.prompt_tokens" as the first key, to reduce the fallback time.
Please adjust the key sequence for all XXX_KEYS constant list.